db_stress_tool/db_stress_test_base.cc

```diff
@@ -346,6 +340,29 @@ bool StressTest::BuildOptionsTable() {
        }},
       {"max_sequential_skip_in_iterations", {"4", "8", "12"}},
   };
+  if (FLAGS_compaction_style == kCompactionStyleUniversal &&
+      FLAGS_universal_max_read_amp > 0) {
+    // level0_file_num_compaction_trigger needs to be at most max_read_amp
+    options_tbl.emplace(
+        "level0_file_num_compaction_trigger",
+        std::vector<std::string>{
+            std::to_string(options_.level0_file_num_compaction_trigger),
```

Comment:
assuming options_.level0_file_num_compaction_trigger always <= FLAGS_universal_max_read_amp?

Reply:
Yes, that caused a previous failure and was fixed in #12701